.github/workflows/alpine-unittests.yml

```diff
@@ -53,7 +53,9 @@ jobs:
 
       - name: Create a r/w directory to run tests in
         # without this, tox fails during package install
-        run: lxc exec alpine -- git clone cloud-init-ro/ cloud-init-rw
+        run: |
+          lxc exec alpine -- git config --global --add safe.directory /root/cloud-init-ro//.git
```

Comment:
is the `//` intentional?

Reply:
It was because the first 'fix' failed due the the fact that our git clone provided that trailing `/` in `cloud-init-ro/`. I've dropped the `/` from the git clone and consequently we can drop the `/` from the add safe.directory.